Added features to financial.html

jacobxcodes opened the pull request:
Added list of bank accts, money laundering businesses, blackmailed officials, & list of businesses

```
git fetch -all
git checkout financials

```
1. Added div with class=cards_large
1. Created two divs inside with class=cards_split
1. Added banks accounts and associated businesses
1. Created 2nd div with class=cards_large
1. Added blackmailed three blackmailed officials

jacobxcodes committed:
Added list of bank accts, money laundering businesses, blackmailed officials, & list of businesses

jacobxcodes committed:
Same commit as previous with grammar fixed

Levi-Schubert commented:
1. structure needs a div for wrapping all content with the id "content"
1. missing left content box ```<div class = "side__info">```
1. missing the side info nav content

jacobxcodes commented:
1. Added id="content"
1. Created content for class="side_info"
1. Nested id="nav_finance" to previous step
1. Added nav content

``` 
git fetch --all
git checkout financials

```

jacobxcodes committed:
Added side bar nav for finance page. Wrapped content in associated div, class, and id elements

jacobxcodes committed:
Added side bar nav above main content for styling purposes

Levi-Schubert committed:
Merge branch 'master' into financials

Levi-Schubert commented:
👍
